addons/crm/models/crm_lead.py

```diff
@@ -483,14 +485,11 @@ def write(self, vals):
     @api.model
     def search(self, args, offset=0, limit=None, order=None, count=False):
         """ Override to support ordering on activity_date_deadline_my.
-
```

Comment:
Linting: do not remove blank lines ... this makes unnecessary diff and those blank lines help reading ^^ .